Source/WebCore/accessibility/AccessibilityTable.cpp

```diff
@@ -410,6 +412,20 @@ void AccessibilityTable::recomputeIsExposable()
     }
 }
 
+bool AccessibilityTable::isMultiSelectable() const
+{
+    if (!m_isAriaTable)
```

Comment:
This is probably redundant because we have a check for hasGridRole which will be more strict than m_isAriaTable